doc/cascadia/profiles.schema.json

```diff
@@ -465,6 +472,14 @@
               "type": "array"
             }
           ]
+        },
+        "icon": { "$ref": "#/definitions/Icon" },
+        "name": {
+          "description": "The name that will appear in the command palette. If one isn't provided, the terminal will attempt to automatically generate a name.",
```

Comment:
"terminal will attempt to automatically generate a name" if the icon can't be found?

Reply:
No, this description is attached to "name". "icon"'s definition is referenced above.

Reply:
Why is this a thing.